Tests/GitHubRepositories.tests.ps1

```diff
@@ -766,128 +921,232 @@ try
             }
 
-            AfterAll -ScriptBlock {
-                Remove-GitHubRepository -Uri $repo.svn_url -Confirm:$false
+            It "Should return the correct membership" {
+                $repo.owner.login | Should -BeIn $contributors.login
             }
         }
-    }
-
-    Describe 'Contributors for a repository' {
-        BeforeAll {
-            $repo = New-GitHubRepository -RepositoryName ([guid]::NewGuid().Guid) -AutoInit
-        }
 
-        AfterAll {
-            $null = Remove-GitHubRepository -Uri $repo.RepositoryUrl -Confirm:$false
-        }
+        Context 'When getting Github Repository Contributors with Statistics' {
+            BeforeAll {
+                $getGitHubRepositoryContributorParms = @{
+                    OwnerName = $repo.owner.login
+                    RepositoryName = $repoName
+                    IncludeStatistics = $true
+                }
+                $contributors = Get-GitHubRepositoryContributor @getGitHubRepositoryContributorParms
+            }
 
-        Context -Name 'Obtaining contributors for repository' -Fixture {
-            $contributors = @(Get-GitHubRepositoryContributor -Uri $repo.RepositoryUrl)
+            It 'Should return objects of the correct type' {
+                $contributors[0].PSObject.TypeNames[0] | Should -Be 'GitHub.RepositoryContributorStatistics'
+                $contributors[0].author.PSObject.TypeNames[0] = 'GitHub.User'
+            }
 
             It 'Should return expected number of contributors' {
                 $contributors.Count | Should -Be 1
-                $contributors[0].PSObject.TypeNames[0] = 'GitHub.User'
             }
-        }
 
-        Context -Name 'Obtaining contributors for repository (via pipeline)' -Fixture {
-            $contributors = @($repo | Get-GitHubRepositoryContributor -IncludeStatistics)
+            It 'Should return the correct membership' {
+                $repo.owner.login | Should -BeIn $contributors.author.login
+            }
 
-            It 'Should return expected number of contributors' {
-                $contributors.Count | Should -Be 1
-                $contributors[0].PSObject.TypeNames[0] = 'GitHub.User'
+            It 'Should return the correct properties' {
+                $contributors.weeks | Should -Not -BeNullOrEmpty
             }
         }
 
-        Context -Name 'Obtaining contributor statistics for repository' -Fixture {
-            $stats = @(Get-GitHubRepositoryContributor -Uri $repo.RepositoryUrl -IncludeStatistics)
+        Context 'When getting Github Repository Contributors including Anonymous' {
+            BeforeAll {
+                $getGitHubRepositoryContributorParms = @{
+                    OwnerName = $repo.owner.login
+                    RepositoryName = $repoName
+                    IncludeAnonymousContributors = $true
+                }
+                $contributors = Get-GitHubRepositoryContributor @getGitHubRepositoryContributorParms
```

Comment:
This is failing in CI, but only on Windows, and that's because Windows is running under PS5 (intentionally) as opposed to PS7.

You're running into the same problem that @TylerLeonhardt ran into with cc9493204fd3931718864ca7faff7668ec67a787.  You're not collecting this result as an array (e.g. `$contributors = @(Get-GitHubRepositoryContributor @getGitHubReposositoryContributorParms)`), and you're only getting a single result.  That single result won't have a `.Count` property on PS5.  So, you can either do what he did to fix it in the linked CL above, or you can update your test to capture the results as an array.

Reply:
Done